### DISCUSSION TITLE
Could not initialize interactsh client

### DISCUSSION THREAD

**Discussion by www1759896801** [projectdiscovery/nuclei#2123] [General]
[ERR] Could not initialize interactsh client: could not create client: could not make register request: POST https://interact.sh/register giving up after 6 attempts: Post "https://interact.sh/register": context deadline exceeded (Client.Timeout exceeded while awaiting headers)

**Comment by www1759896801** [comment]
how to solve this problem

**Comment by ehsandeep** [comment]
@www1759896801 Update your nuclei engine to latest one which does not make use of `interact.sh` server, the latest version of the project is [v2.7.1](https://github.com/projectdiscovery/nuclei/releases/tag/v2.7.1)

**Reply by www1759896801** [reply]
For example: nuclei -no-interactsh -l /home/kali/xxx.txt ??

**Reply by www1759896801** [reply]
Thank you for your selfless assistance
